packages/db/src/Executions.ts

```diff
@@ -0,0 +1,12 @@
+import Knex from 'knex';
+
+const executionsTableName = 'executions';
+
+export const getDbClient = (knex: Knex) =>
+  knex(executionsTableName);
+
+export const getDbTransaction = (trx: Knex.Transaction) =>
+  trx(executionsTableName);
+
+export const doesExecutionExist = async (params: object, knex: Knex) =>
```

Comment:
I know we discussed keeping the surface of these model "libraries" very thin, but the logic to check for an execution seemed like something we'll do often and not want to rewrite every time

Reply:
Thinking more about this... I'm looking at doing something similar for the asyncOperations table and aside that method, there's no functional difference between what's required. 

I'm loathe to suggest class inheritance here, but perhaps we need the code from getDbClient and getDbTransaction to take a ... config object generator:

e.g. getDbTransaction(executionConfiguration), where the configuration is a object with params like `name`?

and then have only very specific query sets in something like src/executions.ts? 

I don't know that we have to get this spot-on for this, PR, I'm fine on iterating on the approach.

Reply:
I'm not sure I totally understand the proposal. Are we talking about a client/transaction factory of sorts? We can talk about it tomorrow